hazelcast-client/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -1071,9 +1072,10 @@ protected void loadAllInternal(boolean replaceExistingValues, Collection<Data> d
         MapEntrySetCodec.ResponseParameters resultParameters = MapEntrySetCodec.decodeResponse(response);
 
         InflatableSet.Builder<Entry<K, V>> setBuilder = InflatableSet.newBuilder(resultParameters.response.size());
-        SerializationService serializationService = getContext().getSerializationService();
+        InternalSerializationService serializationService
+                = ((InternalSerializationService) getContext().getSerializationService());
         for (Entry<Data, Data> row : resultParameters.response) {
-            LazyMapEntry entry = new LazyMapEntry(row.getKey(), row.getValue(), serializationService);
+            LazyMapEntry entry = new LazyMapEntry(row.getKey(), row.getValue(), serializationService, null);
             setBuilder.add(entry);
```

Comment:
So this entry is never used to extract any attributes from it? The Extractors object does not only provide the ValueExtractors, but it provides the ability to extract value of any property.